alica_engine/src/engine/TeamObserver.cpp

```diff
@@ -73,8 +73,11 @@ bool TeamObserver::updateTeamPlanTrees()
 
     std::vector<AgentId> deactivatedAgentIds;
     bool changedSomeAgent = _tm.updateAgents(deactivatedAgentIds);
```

Comment:
We can get rid of `deactivatedAgentIds` vector completely because it is no longer used anywhere. `updateAgents` can also be updated not to take any parameters